.github/workflows/reusable-release-policy-rego-opa.yml

```diff
@@ -0,0 +1,48 @@
+name: Build and release a Kubewarden policy written in Rego/OPA
+
+on:
+  workflow_call:
+    inputs:
+      oci-target:
+        type: string
+        required: true
+      policy-type:
+        description: "Type of Rego policy; either rego or opa"
```

Comment:
Moreover, this input variable doesn't seem to be used :thinking:

Reply:
Done in https://github.com/kubewarden/github-actions/pull/15/commits/ce9deb84138b523205e4886a5e257fd31c2f95f1.

It gets used on the policies, for example https://github.com/viccuad/disallow-service-loadbalancer-policy/blob/4279ba50ba2b6d0e0461072c2af8beac207b2c96/.github/workflows/release.yml#L29.

Reply:
You are right, it's not being used. sorry, it's on the Makefiles. So we are tying ourselves against the Makefiles of the policies instead of the GH Actions that we used to have on each repository. At least we have funneled the release action..

Moreover, just realized that we indeed have opa and gatekeeper policy templates :facepalm: . Let me update the values for `policy-type` to `opa and `gatekeeper`, and the filenames suffix to `-rego`.